Emby.Server.Implementations/LiveTv/TunerHosts/HdHomerun/HdHomerunUdpStream.cs

```diff
@@ -80,6 +80,10 @@ public override async Task Open(CancellationToken openCancellationToken)
                 }
             }
 
+            if (localAddress.IsIPv4MappedToIPv6) {
```

Comment:
When #4124 and #4125 get accepted this shouldn't be required - as IPv6 will be supported.

Reply:
PR#4125 currently removes all IPv6 addresses as I did read somewhere that not all hd home runs support ipv6.
If the code is accepted with this part - i'll re-test post PR#4125 to ensure nothing breaks.